mkdocs-base.yml

```diff
@@ -112,7 +114,16 @@ plugins:
 extra:
```

Comment:
Update the path to cover_logo in "the with-pdf" section

cover_logo: docs/_static/Percona_Logo_Color.png

Reply:
Done